src/components/Todoapp/components/TodoChanges.vue

```diff
@@ -1 +1,50 @@
-<template>changes made</template>
+<template>
+  <div class="changes">
+    <div class="changes-modify">
+      <TodoButton :buttonName="MarkAll" />
+      <TodoButton :buttonName="ClearCompleted" />
+      <TodoButton :buttonName="DeleteAll" />
+    </div>
+    <div class="changes-display">
+      <TodoButton :buttonName="ShowAll" />
+      <TodoButton :buttonName="ShowActive" class="changes-display-showActive" />
+      <TodoButton :buttonName="ShowCompleted" />
+    </div>
+  </div>
+</template>
+
+<script>
+import TodoButton from "../../../UI/components/TodoButton.vue";
+export default {
+  components: {
+    TodoButton,
+  },
+  data() {
+    return {
+      MarkAll: "Mark All Completed",
+      ClearCompleted: "Clear Completed",
+      DeleteAll: "Delete All",
+      ShowAll: "Show All",
+      ShowActive: "Show Active",
+      ShowCompleted: "Show Completed",
+    };
+  },
```

Comment:
since these are constants we can add these to a constant instead of data since it is reactive